src/circuit.rs

```diff
@@ -987,6 +987,17 @@ mod tests {
             .unzip();
 
         let vk = VerifyingKey::build();
+
+        // Test that the pinned verification key (representing the circuit)
+        // is as expected.
+        {
+            // panic!("{:#?}", vk.vk.pinned());
+            assert_eq!(
+                format!("{:#?}\n", vk.vk.pinned()).as_bytes(),
+                include_bytes!("circuit_description")
```

Comment:
Hmm, could we get this to compare them as strings? Then the output if there is a difference would be more readable. It would be nice if it were also an intelligent diff.